openAi.py

```diff
@@ -0,0 +1,66 @@
+import asyncio
+
+import aiohttp
+import backoff
+import openai
+
+
+class OpenAi:
+    """
+    OpenAIAPI class that uses the OpenAI API to generate explanations.
+    """
+
+    def __init__(self, api_key):
+        """
+        Constructor for OpenAIAPI class.
+        :param api_key: OpenAI API key
+        """
+        self.api_key = api_key
+        self.messages = [
+            {"role": "system", "content": "Please summarize the slides and provide additional information."}
```

Comment:
You give very little context to chatGPT.
Writing prompts is a new art we all need to learn.